test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -461,6 +461,24 @@
                 "51955",
                 "48635",
                 "48644",
+                "53853", # TF 1.15.5, 2.10
+                "53850", # TF 1.15.5, 2.10
+                "53855", # TF 1.15.5, 2.10
+                "53864", # TF 1.15.5, 2.10
+                "53862", # TF 1.15.5, 2.10
+                "53861", # TF 1.15.5, 2.10
+                "53859", # TF 1.15.5, 2.10
+                "53858", # TF 1.15.5, 2.10
+                "53857", # TF 1.15.5, 2.10
+                "53856", # TF 1.15.5, 2.10
+                "53854", # TF 1.15.5, 2.10
+                "53852", # TF 1.15.5, 2.10
+                "53851", # TF 1.15.5, 2.10
+                "53849", # TF 1.15.5, 2.10
+                "53848", # TF 1.15.5, 2.10
+                "53847", # TF 1.15.5, 2.10
+                "53845", # TF 1.15.5, 2.10
+                "53846", # TF 1.15.5, 2.10
```

Comment:
These vulns need to be added to https://github.com/aws/deep-learning-containers/blob/master/data/ignore_ids_safety_scan.json#L64. You can remove this from here altogether in case required, but put it over at the [location](https://github.com/aws/deep-learning-containers/blob/master/data/ignore_ids_safety_scan.json#L64) I mentioned above.

Reply:
Done